package.json

```diff
@@ -11,6 +11,8 @@
     "modules",
     "scripts",
     "prebuilds",
+    "LICENSE",
```

Comment:
npm automatically includes all license files, including those encountered in subdirectories.

Reply:
For reference: https://docs.npmjs.com/cli/v9/configuring-npm/package-json?v=true#files

> Certain files are always included, regardless of settings:
>
> - `package.json`
> - `README`
> - `LICENSE` / `LICENCE`
> - The file in the "main" field

Reply:
fixed for you